controllers/ip-location.js

```diff
@@ -0,0 +1,41 @@
+/*eslint-disable no-process-env*/
+// using process.env here to keep the banner/VPN vars contained – for when the vpn-banner is eventually removed
+
+// IP location data includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com.
+// Test IPs by comparing to the corresponding json, for example: https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoLite2-City-Test.json
+
+"use strict";
+
+require("dotenv").config();
+const maxmindDb = process.env.GEOIP_GEOLITE2_PATH + process.env.GEOIP_GEOLITE2_CITY_FILENAME;
+const Reader = require("@maxmind/geoip2-node").Reader;
+
+async function ipLocation(req, res) {
+  let reader, geoData, countryName, cityName, locationArr;
+  const clientIp = process.env.NODE_ENV === "dev" ? "216.160.83.56" : req.ip; // for dev, return an IP that exists in GeoLite2 test DB, e.g. 216.160.83.56
```

Comment:
Should this use `AppConstants.NODE_ENV`?

Reply:
Yes please. Until we do a better fuller refactor, let's stick to use `AppConstants` consistently.

Reply:
done.